tools/cli/src/main/java/org/apache/carbondata/tool/DataSummary.java

```diff
@@ -314,23 +312,26 @@ private void printColumnStats(String columnName) throws IOException, MemoryExcep
           minPercent = String.format("%.1f", blocklet.getColumnChunk().getMinPercentage() * 100);
           maxPercent = String.format("%.1f", blocklet.getColumnChunk().getMaxPercentage() * 100);
           DataFile.ColumnChunk columnChunk = blocklet.columnChunk;
-          if (columnChunk.column.isDimensionColumn() && DataTypeUtil
+          if (columnChunk.column.hasEncoding(Encoding.DICTIONARY) || blocklet
+              .getColumnChunk().column.getColumnName().contains(".val") || blocklet
```

Comment:
what column is it? can you make it more readable?

Reply:
this will be for no dictionary colmplex column, for complex column min max can be shown as NA, that will be ok right

Reply:
please make correct line wrapping, move `blocklet` to next line

Reply:
done